WordPress/src/test/java/org/wordpress/android/viewmodel/pages/CreatePageListItemActionsUseCaseTest.kt

```diff
@@ -0,0 +1,98 @@
+package org.wordpress.android.viewmodel.pages
+
+import com.nhaarman.mockitokotlin2.mock
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.mockito.junit.MockitoJUnitRunner
+import org.wordpress.android.ui.pages.PageItem.Action.CANCEL_AUTO_UPLOAD
+import org.wordpress.android.ui.pages.PageItem.Action.DELETE_PERMANENTLY
+import org.wordpress.android.ui.pages.PageItem.Action.MOVE_TO_DRAFT
+import org.wordpress.android.ui.pages.PageItem.Action.MOVE_TO_TRASH
+import org.wordpress.android.ui.pages.PageItem.Action.PUBLISH_NOW
+import org.wordpress.android.ui.pages.PageItem.Action.SET_PARENT
+import org.wordpress.android.ui.pages.PageItem.Action.VIEW_PAGE
+import org.wordpress.android.viewmodel.pages.CreatePageUploadUiStateUseCase.PostUploadUiState.UploadWaitingForConnection
+import org.wordpress.android.viewmodel.pages.CreatePageUploadUiStateUseCase.PostUploadUiState.UploadingPost
+import org.wordpress.android.viewmodel.pages.PageListViewModel.PageListType.DRAFTS
+import org.wordpress.android.viewmodel.pages.PageListViewModel.PageListType.PUBLISHED
+import org.wordpress.android.viewmodel.pages.PageListViewModel.PageListType.SCHEDULED
+import org.wordpress.android.viewmodel.pages.PageListViewModel.PageListType.TRASHED
+
+@RunWith(MockitoJUnitRunner::class)
+class CreatePageListItemActionsUseCaseTest {
+    private lateinit var useCase: CreatePageListItemActionsUseCase
+
+    @Before
+    fun setUp() {
+        useCase = CreatePageListItemActionsUseCase()
+    }
+
+    @Test
+    fun `Verify DRAFT actions`() {
+        // Arrange
+        val expectedActions = setOf(VIEW_PAGE, SET_PARENT, PUBLISH_NOW, MOVE_TO_TRASH)
+
+        // Act
+        val draftActions = useCase.setupPageActions(DRAFTS, mock())
+
+        // Assert
+        assertThat(draftActions).isEqualTo(expectedActions)
+    }
+
+    @Test
+    fun `Verify TRASH actions`() {
+        // Arrange
+        val expectedActions = setOf(MOVE_TO_DRAFT, DELETE_PERMANENTLY)
+
+        // Act
+        val trashedActions = useCase.setupPageActions(TRASHED, mock())
+
+        // Assert
+        assertThat(trashedActions).isEqualTo(expectedActions)
+    }
+
+    @Test
+    fun `verify that SCHEDULED & PUBLISHED actions are the same`() {
```

Comment:
Yes, thanks for pointing this out. I would say that the actions being the same aren't necessarily a coincidence since a `Scheduled` page is just a page being `Published` at a later date 😄 Nonetheless, it's still good to have them separate since the business requirements could be different for both so I resolved this here 4e6dc61